main.tf

```
@@ -4,16 +4,24 @@
 # We might add more than one line for additional information
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-resource "google_logging_project_sink" "sink" {
-  count = var.module_enabled ? 1 : 0
+locals {
+  create_project_sink      = var.folder == null && var.organization == null
+  create_folder_sink       = var.folder != null && var.organization == null && var.project == null
+  create_organization_sink = var.organization != null && var.folder == null && var.project == null
```

sorry i proposed wrong stuff:

```suggestion
  create_project_sink      = var.project != null || (var.folder == null && var.organization == null)
  create_folder_sink       = var.project == null && var.folder != null
  create_organization_sink = var.project == null && var.folder == null && var.organization != null
```